src/components/ModalEditDialog/ModalEditDialog.jsx

```diff
@@ -6,7 +6,7 @@ import useForm from '../../hooks/useForm';
 
 const rootSelector = document.getElementById('root');
 
-const ModalTableEditDialog = ({ dataObject, columnsConfig, onClose, onUpdateData }) => {
+const ModalEditDialog = ({ dataObject, columnsConfig, onClose, onUpdateData }) => {
```

Comment:
Since we use `Dialog` word in the name, let's avoid `Modal` word and I guess better to have Table in the name, since this dialog we did for the table